pkg/harvester/types.ts

```diff
@@ -0,0 +1,26 @@
+export const HCI = {
+  VM:                 'kubevirt.io.virtualmachine',
+  VMI:                'kubevirt.io.virtualmachineinstance',
+  VMIM:               'kubevirt.io.virtualmachineinstancemigration',
+  VM_TEMPLATE:        'harvesterhci.io.virtualmachinetemplate',
+  VM_VERSION:         'harvesterhci.io.virtualmachinetemplateversion',
+  IMAGE:              'harvesterhci.io.virtualmachineimage',
```

Comment:
`harvesterhci.io.virtualmachineimage` is used in the harvester machine config and within the standalone harvester UI... so I guess it would need to be defined in two places?

It might be worth noting that a user could be provisioning an rke2 cluster with the harvester driver without having imported a harvester cluster, via the external cloud credential option (I think?).

Reply:
Interesting find! Took a while to dig through this.

I've updated `harvester/findAll` to use `cluster/request` which is inline with the other actions. Using `findAll` would associate the namespaces with whatever was the current dashboard cluster context rather than the one in the url). 
https://github.com/rancher/dashboard/blob/610957374034d5252cc2560eb524ece701d79598/shell/machine-config/harvester.vue#L102 

As all of these actions are `/request` they bypass any classify / convert to model process, so no need to duplicate the virtualmachineimage model in shell. If they did use find/findAll though we could have had a shallow clone of the plugin's model in `shell/models/cluster/harvesterhci.io.virtualmachineimage` (we would need this in the `cluster` folder to make sure it was loaded over any plugin model)

Reply:
Noice, we definitely don't want to have to maintain a dupe of a resource model. A considerably smaller issue remains:  `HCI.IMAGE` is undefined here; we need to bring that type back into shell/config/types (or, eventually, duplicate it in both the harvester manager plugin and the harvester plugin).

Aside from this small change the PR looks good!